scm/etc/Cheyenne_setup_gnu.sh

```diff
@@ -6,7 +6,7 @@ echo "Setting environment variables for SCM-CCPP on Cheyenne with gcc/gfortran"
 echo "Loading gnu and netcdf modules..."
 module purge
 module load ncarenv/1.3
-module load gnu/8.3.0
+module load gnu/9.1.0
```

Comment:
Once #240 gets merged, the Cheyenne setup scripts will be set to use GNU 10. Is that OK for RT purposes?

Reply:
GNU 10. should be fine. v8.3.0 was having issues with the findloc statements in rrtmgp_sampling.F90

Reply:
Yep, which is one reason why GNU was upgraded for the release.